nixos/modules/services/networking/tailscale.nix

```diff
@@ -29,6 +29,18 @@ in {
       description = lib.mdDoc "Username or user ID of the user allowed to to fetch Tailscale TLS certificates for the node.";
     };
 
+    enableLogUpload = mkOption {
```

Comment:
Can we call this DisableLogsNoSupport or something like that? Not everyone reads the description so it would be good to get the trade-off across in the name as well.

Reply:
I would not name it as something that discourages users to toggle it out of fear.
Practically speaking, Tailscale will not refuse support, they will just ask you to turn on logging before they can help you further. To have a look into your logs, they will ask you for your machine identifier, which you generate using `tailscale bugreport`. If you have logs turned off, the command outputs `BUG-NO-LOGS-NO-SUPPORT-this-node-has-had-its-logging-disabled` so its relatively clear whats going on, you cannot really miss it.

Reply:
Yepp that’s fair!